smithy-model/src/test/resources/software/amazon/smithy/model/errorfiles/validators/resource-operation-errors.errors

```diff
@@ -0,0 +1,2 @@
+[ERROR] smithy.example#Resource: Resource property `price` is not used in the input or output of create or an instance operation. | ResourceOperationInputOutput
+[NOTE] smithy.example#Price: The string shape is not connected to from any service shape. | UnreferencedShape
```

Comment:
Oh interesting. Properties should contribute Relationships froma  resource to their properties. Identifiers do this already. We need to make sure that's working and also specified in the selectors specification around neighbors.